virtool/analyses/api.py

```diff
@@ -242,7 +251,7 @@ async def get(self, upload_id: int, /) -> Union[r200[FileResponse], r404]:
         """
         Download a file.
 
-        Downloads a file associated with an analysis. Some workflows retain key files
+        Fetches the file linked to an analysis. Some workflows retain key files
```

Comment:
This should not change as the file is actually downloaded instead of a JSON representation being received.